<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EllipseDetectionDialog</class>
 <widget class="QDialog" name="EllipseDetectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ellipse Detection</string>
  </property>
  <property name="windowOpacity">
   <double>0.800000000000000</double>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>571</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1">
     <widget class="QLabel" name="MinDiameter">
      <property name="text">
       <string>Min Diameter</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="4">
     <widget class="QCheckBox" name="BlackWhiteBox">
      <property name="text">
       <string>Activate BlackWhite</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QSlider" name="thresholdSlider">
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="2">
     <widget class="QSlider" name="horPosition">
      <property name="value">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QCheckBox" name="activateBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Activate Processing</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="2" colspan="4">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>                                                                    Fit parameters</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" rowspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Aspect Ratio</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>                  Threshold</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" rowspan="2">
     <widget class="QCheckBox" name="IntervalleDiameter">
      <property name="text">
       <string>Activate Diameter Intervalle</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="2">
     <widget class="QSlider" name="vertPosition">
      <property name="pageStep">
       <number>10</number>
      </property>
      <property name="value">
       <number>80</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0" rowspan="2">
     <widget class="QCheckBox" name="AspectRatioBox">
      <property name="text">
       <string>Activate Aspect Ratio</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="MaxDiameter">
      <property name="text">
       <string>Max Diameter</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2" rowspan="2" colspan="2">
     <widget class="QSlider" name="AspectRatioSlider">
      <property name="maximum">
       <number>99</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>80</number>
      </property>
      <property name="sliderPosition">
       <number>80</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="9" column="1" rowspan="2" colspan="3">
     <widget class="QPushButton" name="trackButton">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Enable Feedback</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="0" rowspan="2">
     <widget class="QCheckBox" name="ContoursIntervalleBox">
      <property name="text">
       <string>Activate Contours Intervalle</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
